src/main/java/com/btk5h/skriptmirror/skript/CondAcceptsChange.java

```diff
@@ -0,0 +1,123 @@
+package com.btk5h.skriptmirror.skript;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.UnparsedLiteral;
+import ch.njol.skript.util.Patterns;
+import ch.njol.skript.util.Utils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+
+public class CondAcceptsChange extends Condition {
+
+  private static final Patterns<ChangeMode> PATTERNS = new Patterns<>(new Object[][] {
+      {"%classinfo% can be added to %expressions%", ChangeMode.ADD},
+      {"%classinfo% (can't|cannot) be added to %expressions%", ChangeMode.ADD},
```

Comment:
```suggestion
      {"%classinfo% can be (added|given) to %expressions%", ChangeMode.ADD},
      {"%classinfo% (can't|cannot) be (added|given) to %expressions%", ChangeMode.ADD},
```
Not sure if it's worth doing this, but if you agree, the other modes should be done as well, like `clear` for DELETE

Reply:
sounds like a good idea. i can't recall if skript has a `dirt can be given to player` type syntax though. i'll have to double check so we dont accidentally cause a conflict if there is